dev-tools/export_dashboards.py

```diff
@@ -119,7 +115,7 @@ def main():
     args = parser.parse_args()
 
     if args.index is None:
-        args.index = args.beat.lower() + "-*"
+        args.index = args.beat.lower()
```

Comment:
Actually with this you found the bug that the index-pattern should be stored under /.kibana/index-pattern/metricbeat-* (and not metricbeat). This way, the name of the index pattern in Kibana is `metricbeat-*`, instead of `metricbeat`. This issue was available in the old script, and I copied in the new script as well. I opened a [PR](https://github.com/elastic/beats/pull/2348) to fix it in `import_dashboards.go`.